src/test/java/uk/gov/companieshouse/fixtures/DissolutionFixtures.java

```diff
@@ -138,4 +137,28 @@ public static DirectorApproval generateDirectorApproval() {
 
         return approval;
     }
+
+    public static List<CompanyOfficerApi> generateCompanyOfficerList() {
+        final CompanyOfficerApi officerOne = new CompanyOfficerApi();
+
+        officerOne.setName("John Doe");
+        officerOne.setAppointedOn(LocalDateTime.now().minusWeeks(1).toLocalDate());
+
+        final CompanyOfficerApi officerTwo = new CompanyOfficerApi();
+
+        officerTwo.setName("Fred Mercure");
+        officerTwo.setAppointedOn(LocalDateTime.now().minusWeeks(1).toLocalDate());
+
+        return Arrays.asList(officerOne, officerTwo);
+    }
+
+    public static CompanyProfileApi generateCompanyProfileApi() {
+        final CompanyProfileApi companyProfileApi = new CompanyProfileApi();
+        companyProfileApi.setCompanyName("My Company");
+        companyProfileApi.setType(CompanyType.LTD.getValue());
+        companyProfileApi.setCompanyNumber("10001");
+        companyProfileApi.setCompanyStatus(CompanyStatus.ACTIVE.getValue());
+
+        return companyProfileApi;
+    }
```

Comment:
I don't think these should be in the DissolutionFixtures tbh - they have nothing to do with dissolution-specific stuff. Instead, we should have a separate CompanyFixtures and CompanyOfficerFixtures imo